src/Tizen.NUI.Components/Controls/Popup.cs

```diff
@@ -738,17 +738,19 @@ public override void ApplyStyle(ViewStyle viewStyle)
                     ButtonHeight = (int)ppStyle.Buttons.SizeHeight;
                 }
 
-                buttonStyle = (ButtonStyle)ppStyle.Buttons?.Clone();
```

Comment:
It seems that this variable is not local variable.

Reply:
Thank you. I renamed the variable!